src/shader/shaderLightSampling.js

```diff
@@ -0,0 +1,87 @@
+export const shaderLightSampling = /* glsl */`
+
+struct LightSampleRec {
+	bool hit;
+	float dist;
+	vec3 direction;
+	float pdf;
+	vec3 emission;
+};
+
+LightSampleRec lightsClosestHit( sampler2D lights, uint lightCount, vec3 rayOrigin, vec3 rayDirection ) {
+
+	LightSampleRec lightSampleRec;
+	lightSampleRec.hit = false;
```

Comment:
Should we initialize all the struct values in case we don't hit any light front faces?